packages/editor/src/nodes/GroundPlaneNode.ts

```diff
@@ -70,5 +72,8 @@ export default class GroundPlaneNode extends EditorNodeMixin(GroundPlane) {
       receive: this.receiveShadow,
       cast: false
     })
+    this.addGLTFComponent('navmesh', {
```

Comment:
this is not needed, generateNavmesh will be serialised into `ground-plane` and `navmesh` has no deserialization in either the location nor the editor